Kitodo-API/src/main/java/org/kitodo/api/dataeditor/rulesetmanagement/SimpleMetadataViewInterface.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataeditor.rulesetmanagement;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Provides an interface for the meta-data key view service. The meta-data key
+ * view service provides a filtered view on meta-data keys.
+ */
+public interface SimpleMetadataViewInterface extends MetadataViewInterface {
+    /**
+     * Maps a boolean value to a meta-data value.
+     *
+     * @param value
+     *            boolean input value
+     * @return value to save as a meta-data entry. If absent, delete the
+     *         meta-data entry.
+     */
+    default Optional<String> convertBoolean(boolean value) {
+        return value ? Optional.of(getSelectItems().entrySet().iterator().next().getKey()) : Optional.empty();
+    }
+
+    /**
+     * Returns the default value for the input type boolean.
+     *
+     * @return the default value
+     */
+    default boolean getBooleanDefaultValue() {
```

Comment:
A default value, which depends on a condition?

Reply:
Yes exactly. Either the default strings contain a value, then it's `true`, if not it's `false`. The value should then be the value that, if `true`, is also written to the meta-dataₜₒₜₐₗᵢₜᵧ.